docs/kyma-environment-broker/03-03-hyperscaler-account-pool.md

```diff
@@ -0,0 +1,25 @@
+# Hyperscaler Account Pool
```

Comment:
Just an idea - this seems to be quite important, many components interact - maybe (some day) add a simple diagram to illustrate what calls what and what is stored where?